src/app/routes/features/features.component.ts

```diff
@@ -12,13 +15,29 @@ import MapUtils from '../../shared/utils/map.utils';
 export class FeaturesComponent implements OnInit {
 
   features: Feature[];
+  gridApi: GridApi;
+  columnApi: ColumnApi;
+  gridOptions: GridOptions;
+  rowData: Feature[];
 
   constructor(private featureService: FeatureService, private logger: NGXLogger) {
+      this.gridOptions = {
+          headerHeight: 0,
+          columnDefs: [],
+          rowHeight: 160,
+          suppressHorizontalScroll: true,
+          fullWidthCellRenderer: 'fullWidthCellRenderer',
+          frameworkComponents: {
+              fullWidthCellRenderer: FeatureRendererComponent
+          },
+          isFullWidthCell: (rowNode: RowNode) => true
+      };
   }
 
   ngOnInit() {
     this.featureService.getFeatures().subscribe((response) => {
       this.features = response;
+      this.rowData = response;
```

Comment:
Any reason why we need this.features and this.rowData. Both seem to be the household for the same.

Reply:
It is needed for the commented code. As I ll remove the commented code, I ll refactor this part as well.